.github/labeler.yml

```diff
@@ -1,104 +1,69 @@
+"area: crates":
+  - "crates/**/*"
+
 "area: dependencies":
-  - "**/patches/*"
-  - "**/patches/.*"
   - "**/patches/**/*"
-  - "**/patches/**/.*"
   - "**/yarn.lock"
 
 "area: hub":
-  - "hub/*"
-  - "hub/.*"
   - "hub/**/*"
-  - "hub/**/.*"
 
 "area: infra":
-  - ".github/*"
-  - ".github/.*"
+  - ".changeset/config.json"
   - ".github/**/*"
-  - ".github/**/.*"
-  - ".husky/*"
-  - ".husky/.*"
-  - "scripts/*"
-  - "scripts/.*"
+  - ".husky/**/*"
+  - "infra/**/*"
   - "scripts/**/*"
-  - "scripts/**/.*"
   - "*"
-  - ".*"
 
 "area: packages":
-  - "packages/*"
-  - "packages/.*"
   - "packages/**/*"
-  - "packages/**/.*"
+
+"area: rfcs":
+  - "rfcs/**/*"
 
 "area: site":
-  - "site/*"
-  - "site/.*"
   - "site/**/*"
```

Comment:
We should make `site` a sub-folder within `apps`

Reply:
That's in scope for https://github.com/blockprotocol/blockprotocol/pull/609 (see TODO in PR description)